applications/callflow/doc/webhook.md

```diff
@@ -1,14 +1,22 @@
-## Webhook
-
-### About Webhook
+# Webhook
 
 Send a custom webhook to your web server during the callflow.
 
-#### Schema
+- [The Webhook Callflow Action](#orgec83ef6)
+  - [The Webhook action](#orga648329)
+  - [Using the webhook action in a callflow](#orged8a935)
+
+
+<a id="orgec83ef6"></a>
+# The Webhook Callflow Action
+
+Webhooks can be triggered from a callflow without needing them to be predefined by an API call. They are useful in tracking the state of a caller in a phone tree, triggering actions on the receiver's end, or whatever imagination can come up with.
 
-Validator for the webhook callflow data object
 
+<a id="orga648329"></a>
+## Schema
 
+See the [schema](https://github.com/2600hz/kazoo/blob/master/applications/crossbar/priv/couchdb/schemas/callflows.webhook.json) for details of what can go in the action's data object.
```

Comment:
@harenson These documents for `Data` usage inside callflow module have a reference file in `applications/callflow/doc/ref/*`

These reference files are generating automatically from their schema (in `applications/crossbar/priv/couchdb/schemas/callflow.*` by `kazoo_ast/cf_data_usage ` as part if `make apis` target.

So if you may please use that ref doc and merge in the content of `webhooks.md`, so the schema table is shown here.